### PR TITLE
Add physical network type: cascading to openstack helpers

### PR DESCRIPTION
**Pull request by sab-systems**
OpenTelekom Cloud is based on cascading OpenStack.
The type of network interfaces used for KVM instances 
here is:'cascading'.

This commit adds 'cascading' to the list of KNOWN_PHYSICAL_TYPES.

### CODE REVIEW

**Comment by OddBloke** [comment]
Hi @sab-systems,

Thanks for the contribution to cloud-init! Before we can land this, we need you to sign the Canonical CLA, which you can find [here](https://ubuntu.com/legal/contributors). Please be sure to include your GitHub username in the submission!


Thanks,

Dan

**Comment by sab-systems** [comment]
Hi @OddBloke,

thanks for link to Canonical CLA.
Signed it today.

Sabrina

**Comment by OddBloke** [comment]
Hey Sabrina, did you get a chance to look at the test that Scott suggested adding to this proposal?

**Comment by sab-systems** [comment]
Finally had a look on recommended helpers test from Scott:
./cloudinit/sources/helpers/tests/test_openstack.py.
Run this unittest and played a little bit with different configs. 
The test works for me.

Shall we include it into this pull request?

**Comment by smoser** [comment]
> Shall we include it into this pull request?

Yes please.

File: cloudinit/sources/helpers/tests/test_openstack.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                 'type': 'physical'},
+                {'address': '8.8.8.8', 'type': 'nameserver'}]}
+
+        for t in openstack.KNOWN_PHYSICAL_TYPES:
+            net_json["links"][0]["type"] = t
+            self.assertEqual(
+                expected,
+                openstack.convert_net_json(network_json=net_json,
+                                           known_macs=macs))
+
```
Comment (OddBloke) [inline cloudinit/sources/helpers/tests/test_openstack.py]:
```suggestion
```

**Comment by OddBloke** [comment]
On Tue, Feb 25, 2020 at 06:23:09AM -0800, Scott Moser wrote:
> smoser approved this pull request.
> 
> The error above is just 
> 
>     cloudinit/sources/helpers/tests/test_openstack.py:45:1: W391 blank line at end of file
> 
> You'll need to fix that.  I'll approve, but it can't land anyway until the c-i bot does so as well.

I've just used my maintainer powers to remove this line.  I've merged
master, so hopefully we'll get a green CI run and be able to land this
shortly.

**Pull request merged**